stdlib/public/core/PrefixWhile.swift

```diff
@@ -98,15 +98,15 @@ extension LazySequenceProtocol {
   }
 }
 
-/// A lazy `${Collection}` wrapper that includes the initial consecutive
+/// A lazy collection wrapper that includes the initial consecutive
 /// elements of an underlying collection that satisfy a predicate.
 ///
-/// - Note: The performance of accessing `endIndex`, `last`, any methods that
-///   depend on `endIndex`, or moving an index depends on how many elements
+/// - Note: The performance of accessing `endIndex`, `last`, or any methods that
+///   depend on `endIndex`, or of moving an index, depends on how many elements
```

Comment:
This is still difficult to follow.  Perhaps something like:

```
`endIndex, `last`, moving an index, or any methods that depend on `endIndex` depends on how
```

would be better?